Test/SqlDom/ParserErrorsTests.cs

```diff
@@ -6950,5 +6950,98 @@ Description NVARCHAR(200)
                                             ";
             ParserTestUtils.ErrorTestFabricDW(identityColumnSyntax2, new ParserErrorInfo(identityColumnSyntax2.IndexOf("IDENTITY(") + 8, "SQL46010", "("));
         }
+
+        /// <summary>
+        /// Negative tests for VECTOR INDEX syntax
+        /// </summary>
+        [TestMethod]
+        [Priority(0)]
+        [SqlStudioTestCategory(Category.UnitTest)]
+        public void VectorIndexNegativeTests()
+        {
+            // Missing VECTOR keyword
+            ParserTestUtils.ErrorTest170("CREATE INDEX IX_Test ON dbo.Documents (VectorData)",
```

Comment:
Isn't this valid syntax for normal index creation?

Reply:
Yeah I think the CI tests are failing because of this, you can just remove this test case